<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Сис настройки</string>
        </attribute>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Путь к коллекции</string>
         </property>
        </widget>
        <widget class="QToolButton" name="toolButton">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>40</y>
           <width>25</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Last.FM</string>
        </attribute>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Скроблить</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Имя пользователя</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Пароль пользователя</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
